README.md

```diff
@@ -48,6 +48,12 @@ Requires `node 16.18`
 npm install
 ```
 
+### Create app config
+Create an app.config.js which can be edited for custom settings.
+```
+cp src/config/app.config.example.js ./src/config/app.config.js
```

Comment:
cp src/config/app.config.example.js src/config/app.config.js

Reply:
please remove "./" from the destination path for consistency @rain64x

Reply:
Sure, thank you for pointing it out @mareksmakosz!